Source/Scene/Gltf3DTileContent.js

```diff
@@ -46,15 +27,8 @@ function Gltf3DTileContent(tileset, tile, resource, gltf) {
   this._tile = tile;
   this._resource = resource;
   this._model = undefined;
-  this._batchTable = undefined;
-  this._features = undefined;
-
-  this._classificationType = tileset.vectorClassificationOnly
-    ? undefined
-    : tileset.classificationType;
 
   // Populate from gltf when available
-  this._batchIdAttributeName = undefined;
   this._diffuseAttributeOrUniformName = {};
 
   this.featurePropertiesDirty = false;
```

Comment:
This can be removed

Reply:
No, that's part of the `Cesium3DTileContent` API

Reply:
Ah right!